pkg/profileparser/profileparser.go

```diff
@@ -822,6 +822,14 @@ func newStandardParser() *referenceParser {
 	if pcidssperr != nil {
 		log.Error(nciperr, "Could not register PCI-DSS reference parser") // not much we can do here..
 	}
+	stigperr := p.registerStandard("STIG", `^https://public\.cyber\.mil/stigs/downloads/\?_dl_facet_stigs=operating-systems%2Cunix-linux$`)
+	if stigperr != nil {
+		log.Error(nciperr, "Could not register STIG reference parser") // not much we can do here..
+	}
+	srgperr := p.registerStandard("SRG-APP-CTR",`^https://public\.cyber\.mil/stigs/downloads/\?_dl_facet_stigs=container-platform$`)
+	if srgperr != nil {
+		log.Error(nciperr, "Could not register SRG-APP-CTR reference parser") // not much we can do here..
```

Comment:
I could see where someone reading the logs might get confused by what this means compared to STIG one above.

If they're both related to the STIG, could we use the same error message?

Reply:
Oh may, you are totally correct.
Both the STIG ID `CNTR-OS-XXXXXX` and container SRG `SRG-APP-XXXXXX-CTR-XXXXXX` are found at https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=container-platform

Reply:
With same URI for both the STIGIDs and SRGs, the references will be listed together under the same name `STIG` :
```
$oc get rule  ocp4-api-server-tls-security-profile -o=jsonpath={.metadata.annotations} | jq -r
{
  "compliance.openshift.io/image-digest": "pb-ocp4jm9bc",
  "compliance.openshift.io/profiles": "ocp4-nerc-cip,ocp4-moderate-rev-4,ocp4-high-rev-4,ocp4-high,ocp4-moderate,ocp4-stig-v1r1,ocp4-stig",
  "compliance.openshift.io/rule": "api-server-tls-security-profile",
  "control.compliance.openshift.io/NIST-800-53": "SC-8;SC-8(1)",
  "control.compliance.openshift.io/STIG": "SRG-APP-000014-CTR-000040;SRG-APP-000560-CTR-001340;CNTR-OS-000020",
  "policies.open-cluster-management.io/controls": "SC-8,SC-8(1),SRG-APP-000014-CTR-000040,SRG-APP-000560-CTR-001340,CNTR-OS-000020",
  "policies.open-cluster-management.io/standards": "NIST-800-53,STIG"
}
```

`  "control.compliance.openshift.io/STIG": "SRG-APP-000014-CTR-000040;SRG-APP-000560-CTR-001340;CNTR-OS-000020",`